src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -331,6 +331,10 @@ def load_arguments(self, _):
         c.argument('sku', arg_type=sku_arg_type)
         c.argument('dryrun', help="show summary of the create and deploy operation instead of executing it", default=False, action='store_true')
 
+    with self.argument_context('webapp ssh') as c:
+        c.argument('slot', options_list=['--slot', '-s'], help='slot to restore the deleted content to')
+        c.argument('container', options_list=['--container', '-c'], help='the DNS name of container in multi-container web app (currently not supported)')
```

Comment:
What exactly is not currently supported? It almost sounds like the parameter is not supported.

Reply:
yes, its not supported yet, ill remove it